probe/endpoint/resolver_test.go

```diff
@@ -0,0 +1,40 @@
+package endpoint_test
+
+import (
+	"testing"
+
+	"github.com/weaveworks/scope/probe/endpoint"
+	"time"
+)
+
+func TestReverseResolver(t *testing.T) {
+	revRes := endpoint.NewReverseResolver(100)
+	tests := []struct {
+		ip   string
+		name string
+	}{
+		{"8.8.8.8", "google-public-dns-a.google.com."},
+		{"8.8.4.4", "google-public-dns-b.google.com."},
```

Comment:
This unit test will fail if the host isn't internet-connected, or if Google changes their DNS infrastructure.

I suppose a good way to fix this would be to mock out net.LookupAddr, and swap it at the start of the test.